Userland/Games/Snake/Game.cpp

```diff
@@ -66,7 +66,7 @@ ErrorOr<NonnullRefPtr<Game>> Game::try_create()
     }
 
     auto color = Color::from_argb(Config::read_u32("Snake"sv, "Snake"sv, "BaseColor"sv, Color(Color::Green).value()));
-    auto skin_name = TRY(String::from_byte_string(Config::read_string("Snake"sv, "Snake"sv, "SnakeSkin"sv, "Classic"sv)));
```

Comment:
Why change the config key from "SnakeSkin" to "ImageSkin"?

Reply:
My misunderstanding the difference between the different types of skins. Changed back :)